architecture/storage-layer.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Architecture&#58; Storage Layer
+summary: 
+toc: false
+---
+
+The Storage Layer of CockroachDB's architecture reads and writes data from disk.
```

Comment:
Later you use `read and write data to disk`. Perhaps we should use `to` here as well, or the more complete but clunky `reads data from and writes data to disk`.